docs/community/release_notes/release_note_0.3.0.md

```diff
@@ -0,0 +1,368 @@
+---
+hide:
+  - navigation
+
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this line format is broken (actually many files are broken ...)

Reply:
ok. this line prefix shold fill some spaces.